microceph/database/schema.go

```diff
@@ -46,3 +47,21 @@ CREATE TABLE services (
 
 	return err
 }
+
+// Adds client config table in database schema.
+func schemaUpdate2(ctx context.Context, tx *sql.Tx) error {
+	stmt := `
+CREATE TABLE client_config (
+  id                            INTEGER  PRIMARY KEY AUTOINCREMENT NOT NULL,
+  member_id                     INTEGER  NOT  NULL,
+  key                           TEXT     NOT  NULL,
+  value                         TEXT     NOT  NULL,
+  FOREIGN KEY (member_id) REFERENCES "internal_cluster_members" (id) ON DELETE CASCADE,
```

Comment:
This likely wouldn't work for the pseudo-host `*` value for a global config

Reply:
As mentioned above, now NULL values are used for global configs.